Umbraco.Community.RuntimeValidators/Umbraco.Community.RuntimeValidators.csproj

```
@@ -0,0 +1,54 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+	<PropertyGroup>
+		<TargetFramework>net8.0</TargetFramework>
```

Would it be better to target net6.0 to match the net version that Umbraco 10.1.0 uses?
```suggestion
		<TargetFramework>net6.0</TargetFramework>
```